internal/trigger/messagebus/messaging_test.go

```diff
@@ -212,15 +213,16 @@ func TestInitializeAndProcessEventWithOutput(t *testing.T) {
 
 	transform1 := func(edgexcontext *appcontext.Context, params ...interface{}) (bool, interface{}) {
 		transformWasCalled = true
+		fmt.Println("I have been called")
```

Comment:
Do we need these print & log messages now?

Reply:
Nope, just rogue shenanigans.